File: Olebo/src/main/java/jdr/exia/controller/Controller.java
package jdr.exia.controller;

import jdr.exia.model.Model;
import jdr.exia.view.ViewManager;

import javax.swing.text.View;

public class Controller {
    private final ViewManager viewManager;
    private final Model model;

    public Controller(ViewManager viewManager, Model model) {
        this.viewManager = viewManager;
        this.model = model;
    }

    public void initDatas() {
        model.loadDatabase();
    }


    public void start() {

    }
}

File: Olebo/src/main/kotlin/jdr/exia/controller/Controller.kt
package jdr.exia.controller

import jdr.exia.model.dao.DAO
import jdr.exia.view.acts.ActSelector

class Controller {
    fun initDatas(): Array<String> {
        val acts = DAO.getActsList()

        ActSelector(acts).isVisible = true

        return acts
    }

    fun start() {

    }
}
